src/OrchardCore/OrchardCore.DisplayManagement/Title/PageTitleBuilder.cs

```diff
@@ -48,7 +50,7 @@ public IHtmlContent GenerateTitle(IHtmlContent separator)
                 return _fixedTitle;
             }
 
-            if (_title != null)
+            if (_title != null && String.Empty != _title.GetString())
```

Comment:
please use `string.Empty` instead of `String.Empty`

Reply:
If I made this change VS code give me an IDE0049. Can I proceed?

Reply:
Yes. If you rebase your branch, you'll notice the suggestion will go away. It's something we changed in the project after you created your branch